JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -4,11 +4,13 @@ import {
 import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { DxSchedulerModule, DxTemplateModule } from 'devextreme-angular';
-import {
+import dxScheduler, {
   OptionChangedEvent, AppointmentFormOpeningEvent, AppointmentAddingEvent, AppointmentUpdatingEvent,
 } from 'devextreme/ui/scheduler';
 import DataSource from 'devextreme/data/data_source';
 import notify from 'devextreme/ui/notify';
+import { DxSchedulerTypes } from 'devextreme-angular/ui/scheduler';
+import dxForm from 'devextreme/ui/form';
```

Comment:
The same 
https://github.com/DevExpress/devextreme-demos/pull/3074/files#diff-488231ff0b06a9a0bde87310f0888999ac802e000f065eedda96effd409d65deR7